lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -2101,7 +2176,8 @@ export class UserAgentApplication {
             scopes: [this.clientId],
             authority: this.authority,
             account: this.getAccount(),
-            extraQueryParameters: request.extraQueryParameters
+            extraQueryParameters: request.extraQueryParameters,
+            correlationId: request.correlationId
```

Comment:
CorrelationId is not always set by the user. I presume we already are validating and adding this to the request by the time we execute this?

Reply:
updated and cleaned up